terraform/vpc.tf

```diff
@@ -0,0 +1,199 @@
+resource "aws_vpc" "main" {
+    cidr_block = var.vpc-cidr
+    enable_dns_hostnames = true
+    tags = { Name = "tfvpc"}
```

Comment:
no hardcoded values.. make the vpc name a variable, in fact, for generic resources like that it's best practice to have a variable called for example `project` and then you use something like

Name = "${var.project_name}-vpc"

Reply:
@caiocezart agree with you to have a project in var and do something like "${var.project_name}-vpc", but the terraform 1.2 version I am using is using the syntax: var.project_name, I was a bit struggling with how to concatenate  var.project_name with another string, tried a couple of times, will be good if you see any example of this.